azimuth/config.py

```diff
@@ -316,10 +368,27 @@ class AzimuthConfig(
     SimilarityConfig,
     DatasetWarningConfig,
     SyntaxConfig,
+    LanguageConfig,
     extra=Extra.forbid,
 ):
-    # This class should remain empty!
-    pass
+    # Before adding attributes: Remember that dependence on an attribute in AzimuthConfig will
+    # cause a module to include all other configs in its scope.
```

Comment:
I'd change only one word to be clear that it won't automatically "cause" that, but it will "force" you to set the whole `AzimuthConfig` as the module's scope.
```suggestion
    # force a module to include all other configs in its scope.
```

Reply:
```suggestion
    # force a module to include all other configs in its scope.
```
Sounds good! Just going to use "a" instead of "the."

Reply:
The reason why I used "the" was to indicate that I am referring to **_the_** same module as the one with a "dependence on an attribute in AzimuthConfig". I didn't want to sentence to be understood as "dependence on an attribute [...] will force [another random] module to include all other configs in its scope".

Maybe I should have modified the first line
> Remember that if a module depends on an attribute in AzimuthConfig, it will need to include all other configs in its scope.

Reply:
Yes, I understand the intention exactly! When I read what I committed, I read it as exactly that, whereas with "the," the pronoun feels surprising. I'm having trouble figuring out how to explain the grammatical reasoning... 

Currently:
> Before adding attributes: Remember that dependence on an attribute in AzimuthConfig will cause a module to include all other configs in its scope.
It is clear to me that the first phrase ("dependence...AzimuthConfig") belongs to the "a module" mentioned later - that it's a specific module that is the topic (subject?) of the sentence. 

With _the_: 
> Before adding attributes: Remember that dependence on an attribute in AzimuthConfig will cause the module to include all other configs in its scope.
...I'm like, wait, what? Which module were we talking about? _Were_ we talking about a module already?

But, all that said, even if I had a sound grammatical argument, it doesn't matter if it is confusing to you (or any other user). So to be super clear, we could change it to what you suggested, with a slight rephrasing in line with your earlier suggestion, and one of mine to be concise:
> Reminder: If a module depends on an attribute in AzimuthConfig, the module will be forced to include all other configs in its scope.

What do you think? I'm not opposed to doing a new PR for one comment, all in the name of clear communication. :)

Reply:
That's perfect! PR: #310